.github/workflows/ci.yaml

```diff
@@ -26,6 +26,18 @@ jobs:
     - name: build
       run: |
         go build -v ./...
+  buildWithTlsAndSpire:
```

Comment:
Should probably be `without` instead of `with` 😛

Reply:
Yeah sorry will update the pr

Reply:
updated!